File: homework/reverse-a-string.py
# This is my solution for AlgoDaily problem Reverse a String
# Located at https://algodaily.com/challenges/reverse-a-string

def reverse_string(str1):
    # fill this in
   strings=""
   for ch in str1:
      strings = ch + strings
   return strings
reverse=reverse_string("ABCDEF")
print(f"{reverse}")

File: homework/reverse-a-string.js
// This is my solution for AlgoDaily problem Reverse a String
// Located at https://algodaily.com/challenges/reverse-a-string

function reverseString(str){
  // reverse str
  var Arr=str.split('');
  var start=0;
  var end=str.length-1;
  while(start<=end)
  {
    var temp=Arr[start];
    Arr[start]=Arr[end];
    Arr[end]=temp;
    start++;
    end--;
    var revArr=Arr.join('');
   }
  
   return revArr;
}


var reverse=reverseString('ABCEDF');
console.log(reverse);